docs/concepts/faq.md

```diff
@@ -34,6 +35,20 @@ When you add a file to IPFS, IPFS splits it into smaller blocks. IPFS hashes eac
 
 There are a lot of ways you can contribute to IPFS, whether you're interested in helping with either of the core implementations, applications like IPFS Desktop, writing or editing documentation, doing UX, or whatever you enjoy working on. [Get all the details on where to get started here.](../community/contribute/ways-to-contribute.md)
 
+### What is an implementation
+
+An IPFS implementation is any software with basic functionality for interaction with other IPFS implementations. An implementation:
+
+- Supports addressability using CIDs.
+
+- Exposes operations like retrieval, provisioning and indexing on resources using CIDs. The operations that an implementation may support are open-ended, but this requirement should cover any interaction which the implementation exposes to other IPFS implementations.
+
+- Verifies that the CIDs it resolves match the resources they address, at least when it has access to the resources bytes. However, implementations may relax this requirement in controlled environments in which it is possible to ascertain that verification has happened elsewhere in a trusted part of the system.
+
+There are already many IPFS implementations, including [Kubo](../install/command-line.md), [Helia](../install/js-ipfs.md), and [more](../concepts/ipfs-implementations.md).
```

Comment:
I think better to link to the Kubo and Helia project pages.

Also is js-ipfs.md still a thing?